contracts/multisig-prover/src/execute.rs

```diff
@@ -241,10 +241,18 @@ pub fn update_verifier_set(deps: DepsMut, env: Env) -> Result<Response, Contract
                 chain_name: config.chain_name,
             };
 
-            Ok(Response::new().add_submessage(SubMsg::reply_on_success(
-                wasm_execute(config.multisig, &start_sig_msg, vec![])?,
-                START_MULTISIG_REPLY_ID,
-            )))
+            Ok(Response::new()
+                .add_submessage(SubMsg::reply_on_success(
+                    wasm_execute(config.multisig, &start_sig_msg, vec![])?,
+                    START_MULTISIG_REPLY_ID,
+                ))
+                .add_message(wasm_execute(
+                    config.coordinator.clone(),
```

Comment:
The coordinator contract should have its client mod. Check https://github.com/axelarnetwork/axelar-amplifier/blob/24080b4d91528dfab2d91f8ada5d6e0985262be0/contracts/voting-verifier/src/client.rs#L4. Can do in a separate PR.

Reply:
Created a ticket for it.